kratos/includes/serializer.h

```diff
@@ -515,7 +515,7 @@ class KRATOS_API(KRATOS_CORE) Serializer
     void load(std::string const & rTag, array_1d<TDataType, TDimension>& rObject)
     {
         load_trace_point(rTag);
-	//rObject = array_1d<TDataType, TDimension>(); //it generates a warnning --> commented 23/09/2015 <--
+    //rObject = array_1d<TDataType, TDimension>(); //it generates a warnning --> commented 23/09/2015 <--
```

Comment:
leftover?

Reply:
Same: IDE automatically made whitespace changes.

Reply:
remove it unused please